### ISSUE TITLE
Consider pointing to jsch fork

### ISSUE THREAD

**Issue by nivekuil** [clj-commons/clj-ssh#62]
https://github.com/mwiede/jsch is actively updated and has some fixes and features, notably ed25519 keys.

**Comment by xfthhxk** [comment]
@nivekuil I was able to connect and execute `ls` with this [fork](https://github.com/xfthhxk/clj-ssh) but also found other alternatives. [maverick-synergy](https://github.com/sshtools/maverick-synergy) is actively maintained and has a pretty straight forward API to use via interop. [sshj](https://github.com/hierynomus/sshj) might also be of interest.

**Comment by agorgl** [comment]
I bumped today into this issue when trying to connect to a server with ed25519 keys too.
Similar issue seems to exist in gradle: https://github.com/gradle/gradle/issues/19028

**Comment by agorgl** [comment]
Another way to use the fork is putting in your deps.edn:
```
        clj-commons/clj-ssh {:mvn/version "0.5.15" :exclusions [com.jcraft/jsch]}
        com.github.mwiede/jsch {:mvn/version "0.2.8"}}
```

**Comment by slipset** [comment]
Fixed by #68
